products/jbrowse-web/src/ConfigWarningDialog.tsx

```diff
@@ -34,12 +34,10 @@ export default function ConfigWarningModal({
     <Dialog
       open
       maxWidth="xl"
-      aria-labelledby="alert-dialog-title"
```

Comment:
Why get rid of the aria labels for accesibility?

Reply:
I guess this could be restored. I maybe misunderstood their purpose and thought this field was an actual label. That said I feel like our app has pretty poor accessibility already and it is questionable whether retaining small amounts of it helps

Reply:
Removing existing labels won't make it any more accessible 🤷‍♂️

Reply:
restored. it was a misunderstanding by me to remove them